packages/manager/modules/enterprise-cloud-database/src/enterprise-cloud-database.service.js

```diff
@@ -27,32 +26,16 @@ export default class EnterpriseCloudDatabaseService {
     this.OvhApiCloudDBEnterpriseServiceInfos = OvhApiCloudDBEnterprise.ServiceInfos().v6();
     this.OvhApiCloudDBEnterpriseUser = OvhApiCloudDBEnterprise.User().v6();
     this.OvhApiCloudDBEnterpriseOffers = OvhApiCloudDBEnterprise.Offers().v6();
+    this.OvhApiOrderEnterpriseCloudDB = OvhApiOrder.Catalog().Public().cloudDB().v6();
     this.OvhApiMe = OvhApiMe;
-    this.userData = {
-      currencySymbol: null,
-      ovhSubsidiary: null,
-    };
-    this.initialize();
-  }
-
-  initialize() {
-    this.getMe();
   }
 
   getDefaultPaymentMethod() {
     return this.OvhApiMe.PaymentMean().v6().getDefaultPaymentMean();
   }
 
   getMe() {
-    return this.OvhApiMe.v6().get().$promise.then((me) => {
-      this.userData.ovhSubsidiary = me.ovhSubsidiary;
-      this.userData.currencySymbol = me.currency.symbol;
-      return this.data;
-    });
-  }
-
-  getPriceText(priceInCents) {
-    return `${priceInCents / 100000000} ${this.userData.currencySymbol}`;
+    return this.OvhApiMe.v6().get().$promise;
```

Comment:
This should probably be called in the root state of the app and be inherited by the states to avoid multiple API calls for this information